src/utilities/sortByPrice.js

```diff
@@ -0,0 +1,12 @@
+export const sortByPrice = (productList, sortByPrice) => {
+    const ascending = (a, b) => {
+        return a.price.original - b.price.original;
+    };
+
+    const descending = (a, b) => {
+        return b.price.original - a.price.original;
+    };
+    return [...productList].sort(
+        sortByPrice === "ASCENDING" ? ascending : descending
+    );
+};
```

Comment:
This is a very good way of doing it. Instead of writing inside of sort you have just created a callback function for all.